tests/resources/test-stations.json

```diff
@@ -311,6 +329,7 @@
     {
         "testStationId": "19",
         "testStationPNumber": "72-3855852",
+        "testStationStatus": null,
```

Comment:
can you test if this field is empty text instead of explicit null

Reply:
There was a test station with the field not set to test this case in an integration test.
But as Kevin mentioned, I removed the optional field after a discussion with Bogdan.